Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,64 @@
 
 namespace Hazel {
 
+	namespace InstrumentorUtils {
+
+		template <size_t N>
+		struct ChangeResult
+		{
+			char Data[N];
+		};
+
+		template <size_t N>
+		constexpr auto RemoveCdecl(const char(&expr)[N])
+		{
+			ChangeResult<N> result = {};
+
+			size_t srcIndex = 0;
+			size_t dstIndex = 0;
```

Comment:
This is optional, I think this is more preference from me, what do you think?
```suggestion
			size_t srcIndex = 0, dstIndex = 0;
```

Reply:
I am not a fan of declaring multiple variables in the same line.